serious_stuff/lib/screens/HomePage.dart

```diff
@@ -21,8 +21,18 @@ class HomePage extends StatefulWidget {
 }
 
 class _MyHomePageState extends State<HomePage> {
+  bool isMenuLeft = false;
+
+  Future<Null> getSharedPrefs() async {

```

Comment:
I think we should use `Future<void>` instead of `Future<Null>` as the latter is more error prone.